pkg/rpc/request/txBuilder.go

```diff
@@ -195,7 +194,7 @@ func CreateFunctionInvocationScript(contract util.Uint160, params Params) ([]byt
 			if err != nil {
 				return nil, err
 			}
-			emit.String(script.BinWriter, strconv.Itoa(num))
+			emit.Int(script.BinWriter, int64(num))
```

Comment:
What is the problem you're trying to solve, really? This was designed for method name which is a string that can be converted to an integer in transit, you're not supposed to ever hit this for function parameters (`case ArrayT` down below which works fine for integers).

Reply:
Oh, I see now. I have encountered a bug where some contract transfer is not executed because of insufficient funds, but there are other reasons probably.